make/cats

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -o errexit -o nounset
+
+GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
+NAMESPACE="cf"
+
+has_namespace() {
+    kubectl get namespace --output=name "${NAMESPACE}" >/dev/null 2>/dev/null
+}
+
+stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-cats start
+
+if has_namespace ; then
+    :
+else
+    echo 1>&2 Namespace ${NAMESPACE} is missing. SCF is not running.
+    exit 1
+fi
+
+stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-cats::setup start
+
+cf api api.cf-dev.io --skip-ssl-validation
+cf auth admin changeme
+cf enable-feature-flag diego_docker
+
+stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-cats::setup end
+
+stampy "${GIT_ROOT}/scf_metrics.csv" "$0" make-cats::create start
+
+# Delete left-over pod/definition from previous runs, then create/run
+kubectl delete --namespace="${NAMESPACE}" --filename="kube/bosh-task/acceptance-tests.yml"
```

Comment:
The script is failing on the delete for me, because I don't have any prior runs. Add a `2> /dev/null || /bin/true` ?